### PR TITLE
Precision tests

### PR DESCRIPTION
**Pull request by larsgrefer**
This is a demonstration for #21

### CODE REVIEW

File: src/test/java/io/bit3/jsass/PrecisionTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    options.setOutputStyle(OutputStyle.COMPRESSED);
+  }
+
+  private Output getOutput(int precision) throws CompilationException {
+    options.setPrecision(precision);
+    return compiler.compileString(scss, options);
+  }
+
+  @Test
+  public void testLowPrecisionOsX() throws CompilationException {
```
Comment (larsgrefer) [inline src/test/java/io/bit3/jsass/PrecisionTest.java]: This test fails on Linux, but works on OS X

File: src/test/java/io/bit3/jsass/PrecisionTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    Output output = getOutput(5);
+
+    assertThat(output.getCss(), containsString("padding:0 0.8em 0.71429 0.8em"));
+  }
+
+  @Test
+  public void testHighPrecisionOsX() throws CompilationException {
```
Comment (larsgrefer) [inline src/test/java/io/bit3/jsass/PrecisionTest.java]: This test fails on Linux, but works on OS X

File: src/test/java/io/bit3/jsass/PrecisionTest.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    Output output = getOutput(10);
+
+    assertThat(output.getCss(), containsString("padding:0 0.8em 0.7142857143 0.8em"));
+  }
+
+  @Test
+  public void testLowPrecisionLinux() throws CompilationException {
```
Comment (larsgrefer) [inline src/test/java/io/bit3/jsass/PrecisionTest.java]: This test fails on OS X, but works on Linux

File: src/test/java/io/bit3/jsass/PrecisionTest.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    Output output = getOutput(5);
+
+    assertThat(output.getCss(), containsString("padding:0 0.8em .71429 0.8em"));
+  }
+
+  @Test
+  public void testHighPrecisionLinux() throws CompilationException {
```
Comment (larsgrefer) [inline src/test/java/io/bit3/jsass/PrecisionTest.java]: This test fails on OS X, but works on Linux

File: src/test/java/io/bit3/jsass/PrecisionTest.java [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    Output output = getOutput(10);
+
+    assertThat(output.getCss(), containsString("padding:0 0.8em .7142857143 0.8em"));
```
Comment (larsgrefer) [inline src/test/java/io/bit3/jsass/PrecisionTest.java]: This test fails on OS X, but works on Linux

File: src/test/java/io/bit3/jsass/PrecisionTest.java [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    Output output = getOutput(10);
+
+    assertThat(output.getCss(), containsString("padding:0 0.8em 0.7142857143 0.8em"));
+  }
+
+  @Test
+  public void testLowPrecisionLinux() throws CompilationException {
+    Output output = getOutput(5);
+
+    assertThat(output.getCss(), containsString("padding:0 0.8em .71429 0.8em"));
```
Comment (larsgrefer) [inline src/test/java/io/bit3/jsass/PrecisionTest.java]: This test fails on OS X, but works on Linux

File: src/test/java/io/bit3/jsass/PrecisionTest.java [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    Output output = getOutput(5);
+
+    assertThat(output.getCss(), containsString("padding:0 0.8em 0.71429 0.8em"));
+  }
+
+  @Test
+  public void testHighPrecisionOsX() throws CompilationException {
+    Output output = getOutput(10);
+
+    assertThat(output.getCss(), containsString("padding:0 0.8em 0.7142857143 0.8em"));
```
Comment (larsgrefer) [inline src/test/java/io/bit3/jsass/PrecisionTest.java]: This test fails on Linux, but works on OS X

File: src/test/java/io/bit3/jsass/PrecisionTest.java [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  private Output getOutput(int precision) throws CompilationException {
+    options.setPrecision(precision);
+    return compiler.compileString(scss, options);
+  }
+
+  @Test
+  public void testLowPrecisionOsX() throws CompilationException {
+    Output output = getOutput(5);
+
+    assertThat(output.getCss(), containsString("padding:0 0.8em 0.71429 0.8em"));
```
Comment (larsgrefer) [inline src/test/java/io/bit3/jsass/PrecisionTest.java]: This test fails on Linux, but works on OS X

**Comment by tristanlins** [comment]
Closed through #30

**Pull request closed without merging**